components/global-navigation.md

```diff
@@ -26,3 +27,47 @@
 			<a href="#" class="wds-button wds-is-squished wds-global-navigation__start-a-wiki wds-is-secondary">Start a wiki</a>
 		</div>
 	</div>
+
+	<h5>Logged in state</h5>
+	<div class="wds-global-navigation">
+		<div class="wds-global-navigation__content-wrapper">
+			<a href="#" class="wds-global-navigation__logo">
+				<svg class="wds-global-navigation__logo__fandom" alt="Fandom">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
+				</svg>
+				<span class="wds-global-navigation__logo__span">
+					powered by
+				</span>
+				<svg class="wds-global-navigation__logo__wikia" alt="Wikia">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
+				</svg>
+			</a>
+
+			<a href="#" class="wds-global-navigation__content-bar wds-has-fixed-width wds-is-green">Games</a>
```

Comment:
I think we should use `wds-is-games` instead of `wds-is-green`. API will return this class and I don't think it should control the colors.